src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
-'use strict';
+const succsesFunc = (message) => {
+  const successDiv = document.createElement('div');
 
-// write your code here
+  successDiv.className = 'message';
+  successDiv.innerText = message;
+  document.body.appendChild(successDiv);
+};
+
+const errorFunc = (error) => {
+  const errorDiv = document.createElement('div');
+
+  errorDiv.className = 'message error-message';
+  errorDiv.innerText = error;
+  document.body.appendChild(errorDiv);
+};
+
+document.addEventListener('DOMContentLoaded', () => {
+  const promise1 = new Promise((resolve, reject) => {
+    document.querySelector('.logo').addEventListener('click', () => {
+      resolve('Promise was resolved!');
+    });
+  });
+
+  const promise2 = new Promise((resolve, reject) => {
+    setTimeout(() => {
+      // eslint-disable-next-line prefer-promise-reject-errors
+      reject('Promise was rejected!');
+    }, 3000);
+  });
+
+  promise1
```

Comment:
There is no error handling for the case where `document.querySelector('.logo')` returns `null`. If the element with class 'logo' is not found, this will result in a runtime error when trying to add an event listener. Consider adding a check to ensure the element exists before adding the event listener.